configure.ac

```diff
@@ -152,6 +152,10 @@ AC_COMPILE_IFELSE([AC_LANG_PROGRAM([[#define _GNU_SOURCE
 AC_MSG_CHECKING([for feature: NO_NEW_PRIVS])
 AC_COMPILE_IFELSE([AC_LANG_PROGRAM([[
                                      #include <sys/prctl.h>
+                                     #ifndef PR_SET_NO_NEW_PRIVS
```

Comment:
Somethings not right here.  This introduces a regression.  This code block should look like so.

https://github.com/singularityware/singularity/blob/development-2.x/configure.ac#L154-L172

Note the return statements.  I actually don't really understand what is going on here.  Which branch did you use to base this PR?